linalg/hypre.cpp

```diff
@@ -967,17 +974,26 @@ static void MakeWrapper(const hypre_CSRMatrix *mat, SparseMatrix &wrapper)
    wrapper.Swap(tmp);
 }
 
+
```

Comment:
Usually there is exactly one line of whitespace between functions. Please remove this extra empty line.